mage_ai/frontend/components/ContextMenu/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState, useCallback } from 'react';
+
+import FlyoutMenu from '@oracle/components/FlyoutMenu';
+import styled from 'styled-components';
+import { UNIT } from '@oracle/styles/units/spacing';
+import { HEADER_HEIGHT, HEADER_Z_INDEX } from '@components/constants';
+
+export type ContextMenuProps = {
+  children: any;
+  context: ContextMenuEnum;
```

Comment:
You should probably change the name of the `context` prop to something else. Perhaps `location` or something else. Can be confused with React Context or a Nextjs context object.